components/support/jwcrypto/src/lib.rs

```diff
@@ -2,42 +2,67 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-//! Theorically, everything done in this crate could and should be done in a JWT library.
-//! However, none of the existing rust JWT libraries can handle ECDH-ES encryption, and API choices
-//! made by their authors make it difficult to add this feature.
-//! In the past, we chose cjose to do that job, but it added three C dependencies to build and link
-//! against: jansson, openssl and cjose itself.
+//! A library for using Javascript Web Tokens (JWK) and related concepts.
```

Comment:
nit: "Javascript Web Tokens" is "JWT", not "JWK", and actually I don't think we have JWT support in here yet. I would phrase this as "A library for using JSON Object Signing and Encryption (JOSE) data formats such as JWE and JWK" or similar.